iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -54,14 +52,7 @@ using PublisherPortUserType = iox::popo::PublisherPortUser;
 using SubscriberPortUserType = iox::popo::SubscriberPortUser;
 using UniquePortId = popo::TypedUniqueId<popo::BasePortData>;
 
-/// @brief 1 to m communication enabled
-///     for n to m communication use
-///         iox::popo::SubscriberPortMultiProducer
-///         iox::cxx::VariantQueueTypes::SoFi_MultiProducerSingleConsumer;
-using SubscriberPortProducerType = iox::popo::SubscriberPortSingleProducer;
-constexpr cxx::VariantQueueTypes SUBSCRIBER_PORT_QUEUE_TYPE =
-    iox::cxx::VariantQueueTypes::SoFi_SingleProducerSingleConsumer;
-
+using SubscriberPortProducerType = iox::build::CommunicationPolicy;
```

Comment:
Maybe we find a better name? One can get the impression that it is about a producer Type, but the type is a subscriber type. Maybe only SubscriberPortType?

Reply:
You're right done.